src/notebook/index.css

```diff
@@ -37,67 +42,109 @@
 }
 
 
-.jp-MarkdownCell-renderer.p-Widget {
+.jp-MarkdownCell-content.p-Widget {
+  padding-left: 5.6px;
+  padding-top: 7px;
+  padding-right: 7px;
+  padding-bottom: 7px;
   user-select: text;
   -moz-user-select: text;
   -webkit-user-select: text;
   -ms-user-select: text;
 }
 
 
-.jp-MarkdownCell-renderer.p-Widget h1 {
+.jp-MarkdownCell-content.p-Widget h1 {
   font-family: "Helvetica Neue", Arial, Helvetica, Geneva, sans-serif;
   font-size: 26px;
   line-height: 1.0;
+  -webkit-margin-after: 0em;
+  -webkit-margin-before: 0em;
+  -webkit-margin-start: 0em;
+  -webkit-margin-end: 0em;
+  margin-top: 14px; 
 }
 
 
-.jp-MarkdownCell-renderer.p-Widget h2 {
+.jp-MarkdownCell-content.p-Widget h2 {
   font-family: "Helvetica Neue", Arial, Helvetica, Geneva, sans-serif;
   font-size: 22px;
   line-height: 1.0;
+  -webkit-margin-after: 0em;
+  -webkit-margin-before: 0em;
+  -webkit-margin-start: 0em;
+  -webkit-margin-end: 0em;
+  margin-top: 14px; 
```

Comment:
The webkit-margin stuff was being inherited from Chrome's style sheet and was adding some unwanted spacing. We asked over Gitter and were given advice to do it this way, is there another way to reset those margins?